bioconda_utils/pkg_test.py

```diff
@@ -34,10 +29,8 @@ def get_tests(path):
         elif tests_imports and ('perl' in requirements or 'perl-threaded' in requirements):
             tests.append(' && '.join('''perl -e "use %s;"''' % imp for imp in tests_imports))
 
-        tests = ' && '.join(tests)
-        tests = tests.replace('$R ', 'Rscript ')
-    else:
-        pass
+    tests = ' && '.join(tests)
```

Comment:
https://github.com/galaxyproject/galaxy-lib/pull/30
But I think we should hard fail if no test is provided.